lib/ansible/modules/network/mlnxos/mlnxos_lldp.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: mlnxos_lldp
+version_added: "2.5"
+author: "Samer Deeb (@samerd)"
+short_description: Manage LLDP configuration on Mellanox MLNX-OS network devices
+description:
+  - This module provides declarative management of LLDP service configuration
+    on Mellanox MLNX-OS network devices.
+options:
+  state:
+    description:
+      - State of the LLDP protocol configuration.
+    default: present
+    choices: ['present', 'absent']
+"""
+
+EXAMPLES = """
+- name: Enable LLDP protocol
+  mlnxos_lldp:
+    state: present
+
+- name: Disable LLDP protocol
+  mlnxos_lldp:
+    state: lldp
+"""
+
+RETURN = """
+commands:
+  description: The list of configuration mode commands to send to the device
+  returned: always.
+  type: list
+  sample:
+    - lldp
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+from ansible.module_utils.network.mlnxos.mlnxos import BaseMlnxosModule, \
+    show_cmd
```

Comment:
Please do `from ansible.module_utils.network.mlnxos.mlnxos import show_cmd` in next line instead.

Reply:
Done, thanks for the review